packages/amplication-data-service-generator/src/util/preparePackageJsonFile.ts

```diff
@@ -0,0 +1,15 @@
+import { Module } from "@amplication/code-gen-types";
+import { merge } from "lodash";
+
+export function preparePackageJsonFile(
+  module: Module,
+  updateProperties: { [key: string]: any }[]
+): any {
+  const parsedPkg = JSON.parse(module.code);
+  updateProperties &&
+    updateProperties.forEach((updateProperty) =>
+      merge(parsedPkg, updateProperty)
+    );
+
+  return JSON.stringify(parsedPkg, null, 2);
```

Comment:
don't we want to return the module as Module (cade & path) ?
cause at the moment we return only code.

Reply:
The function that is calling this function returns the module with the path and code.